roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -616,6 +620,10 @@ def map_to_pairs(source, delim="="):
     if source == {} or source == '':
         return str()
 
+    # Return early if if input already is a pairs string.
+    if isinstance(source, basestring):
```

Comment:
Use `str` instead of `basestring`

Reply:
This would not work if the input is unicode. See https://stackoverflow.com/questions/1979004/what-is-the-difference-between-isinstanceaaa-basestring-and-isinstanceaaa#answer-1979107.

What would be the reason to ignore and fail on unicode input?

Reply:
Thanks for pointing out unicode.  Maybe it would be better to use `string_types` from `six`?